docs/home_content.md

```diff
@@ -1,3 +1,14 @@
+## Introduction
+
+Orange is providing a full Design System to build Orange Mobile Application. The objective of the [Orange Design System](https://system.design.orange.com/0c1af118d/p/95b685-ios/) (ODS) is to propose a set of guidelines on how to apply th Orange Brand on mobile applications. The Orange design System also provides a series of components and modules that show in details how to use this in the Orange apps.
```

Comment:
There is a typo (coming from in the original text) : `on how to apply th Orange` should be `on how to apply the Orange`.